src/main/controllers/helpers/JudgmentHelpers.ts

```diff
@@ -0,0 +1,511 @@
+import { AppRequest } from '../../definitions/appRequest';
+import { CaseWithId, YesOrNo } from '../../definitions/case';
+import {
+  GenericTseApplicationTypeItem,
+  TseAdminDecisionItem,
+} from '../../definitions/complexTypes/genericTseApplicationTypeItem';
+import { SendNotificationTypeItem } from '../../definitions/complexTypes/sendNotificationTypeItem';
+import { Applicant } from '../../definitions/constants';
+import { applicationTypes } from '../../definitions/contact-applications';
+import { DecisionAndApplicationDetails } from '../../definitions/definition';
+import { HubLinkNames, HubLinkStatus, statusColorMap } from '../../definitions/hub';
+import { AnyRecord } from '../../definitions/util-types';
+
+import { getLanguageParam } from './RouterHelpers';
+
+export const activateJudgmentsLink = (
+  judgmentItems: SendNotificationTypeItem[],
+  decisionItems: TseAdminDecisionItem[],
+  req: AppRequest
+): void => {
+  const userCase = req.session?.userCase;
+  if (
+    (judgmentItems && judgmentItems?.length && judgmentItems !== undefined) ||
```

Comment:
You don't need to test JudgementItems and judgementItems !== undefined.

Also, you're just as fine checking if(judgementItems?.length) because then if judgementItems is undefined, judgementItems?.length will also be undefined and falsy.

Reply:
Do this for decisionItems as well

Reply:
Fixed